core/utilities/mixins/theme/_color-themes.scss

```diff
@@ -1,5 +1,7 @@
 @charset "UTF-8";
 
+@import '../../variables/colors';
```

Comment:
@kgcreative there is a dependency on a variable in this file so I am adding the import declaration in order get stuff to compile.